src/components/addForm/addForm.html

```diff
@@ -0,0 +1,22 @@
+<form>
+  <fieldset>
+    <!--label or legend for description?-->
+    <label for="description">DESCRIPTION</label>
+    <input
+      type="text"
+      name="Enter Task"
+      placeholder="Enter Task"
+      id="addTask"
+    />
+  </fieldset>
```

Comment:
No fieldset needed, just a plain label with input is sufficient!
Be careful though, your `label` is currently not connected with your `input`! The `id` needs to be the same as `for`. And the `name` is usually one word, we'll use it to get the values in JavaScript and it should ideally be named after what the input is, so maybe `description` here as well.
```suggestion
    <label for="description">DESCRIPTION</label>
    <input
      type="text"
      name="description"
      placeholder="Enter Task"
      id="description"
    />
```